music21/stream/base.py

```diff
@@ -11183,10 +11181,7 @@ def makeVoices(self, *, inPlace=False, fillGaps=True):
         # environLocal.printDebug(['makeVoices(): olDict', olDict])
         # find the max necessary voices by finding the max number
         # of elements in each group; these may not all be necessary
-        maxVoiceCount = 1
-        for group in olDict.values():
-            if len(group) > maxVoiceCount:
-                maxVoiceCount = len(group)
+        maxVoiceCount = max([len(group) for group in olDict.values()] + [1])
```

Comment:
This isn't necessarily better than the prior version, so although it's not worth reverting, it might have been worth adding a disable. To keep in mind for future lint additions.

Reply:
I agree, I'm pretty sure there should be a simpler option (maybe `max([..], default=1)`, which would be ok if we know at least one group is not empty), but I wasn't able to confirm it quickly.